public/views/app.devices.html

```diff
@@ -66,4 +66,122 @@ <h5>{{ device.name }}</h5>
+              <div class="col-sm-4 fields-space">
+                <input type="text" class="form-control" ng-model="form.dataItem.schema.unit" placeholder="0" required>
+              </div>
+              <label class="control-label col-sm-2" for="valueType">Value Type*:</label>
+              <div class="col-sm-4 fields-space">
+                <input type="text" class="form-control" ng-model="form.dataItem.schema.valueType" placeholder="3" required>
+              </div>
+              <label class="control-label col-sm-3" for="registerAddress">Register Address*:</label>
+              <div class="col-sm-3 fields-space">
+                <input type="number" class="form-control" ng-model="form.dataItem.modbus.registerAddress" placeholder="1" required>
+              </div>
+              <label class="control-label col-sm-2" for="bitOffset">Bit Offset*:</label>
+              <div class="col-sm-4 fields-space">
+                <select class="form-control" ng-model="form.dataItem.modbus.bitOffset" required>
+                  <option value="1" selected>1</option>
+                  <option value="8">8</option>
+                  <option value="16">16</option>
+                  <option value="32">32</option>
+                  <option value="64">64</option>
+                </select>
+              </div>
+              <label class="control-label col-sm-3" for="lowerThreshold">Lower Threshold:</label>
+              <div class="col-sm-3 fields-space">
+                <input type="number" class="form-control" ng-model="form.config.lowerThreshold" placeholder="1000">
+              </div>
+              <label class="control-label col-sm-3" for="upperThreshold">Upper Threshold:</label>
+              <div class="col-sm-3 fields-space">
+                <input type="number" class="form-control" ng-model="form.config.upperThreshold" placeholder="3000">
+              </div>
+              <label class="control-label col-sm-2" for="change">Change:</label>
+              <div class="col-sm-4 fields-space">
+                <select class="form-control" ng-model="form.config.change">
+                  <option value="yes" selected>Yes</option>
+                  <option value="No">No</option>
+                </select>
+              </div>
+              <label class="control-label col-sm-2" for="timeSec">Time Sec:</label>
+              <div class="col-sm-4 fields-space">
+                <input type="number" class="form-control" ng-model="form.config.timeSec" placeholder="5">
+              </div>
+            </div>
+          </div>
+        </form>
+        <div class="data-item-list" ng-if="currentStep === 'LIST_DATA_ITEMS'">
+          <div class="col-sm-12 flex-center" ng-repeat="dataItem in serviceForm.thingd.dataItems">
+            <div class="row">
+              <div class="col-sm-1 data-item-index">
+                {{ $index + 1 }}
+              </div>
+              <div class="col-sm-6 data-item">
```

Comment:
After removing the index, what do you think about increase the size of this div?

Reply:
Done!